core/src/test/java/org/elasticsearch/common/util/concurrent/ThreadContextTests.java

```diff
@@ -178,6 +184,22 @@ public void testResponseHeaders() {
             threadContext.addResponseHeader("foo", "bar");
         }
 
+        final String now = DateTimeFormatter.RFC_1123_DATE_TIME.format(ZonedDateTime.now(ZoneId.of("GMT")));
+        final Function<String, String> deduplicator = v -> {
```

Comment:
should we make this a static value somewhere in DeprecationLogger to make sure we test it as well?